Changelog.md

```diff
@@ -7,6 +7,12 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 106 - UNRELEASED
+
+### Added
+
+-   Nrfutil device `uart-write` support
```

Comment:
Could you add some more description to what this feature does?
E.g. why do we want to add it, what is it's main purpose and are there other use cases?